subsys/fs/nvs/nvs.c

```diff
@@ -149,7 +150,7 @@ static int _nvs_flash_block_cmp(struct nvs_fs *fs, u32_t addr, const void *data,
 		}
 		len -= bytes_to_cmp;
 		addr += bytes_to_cmp;
-		data += bytes_to_cmp;
+		data = (const u8_t *)data + bytes_to_cmp;
```

Comment:
shouldn't use `data8` here too ?

Reply:
Good catch. I'll fix to a consistent style.